thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/route.js

```diff
@@ -42,9 +39,64 @@ export default Ember.Route.extend({
     this._super(...arguments);
     if (isExiting) {
       controller.clearAll();
+      requestCanContinue = false;
```

Comment:
Is the intention here to stop the checkReplayStatus loop on exit? 
If that's the case, I feel it'd be cleaner to implement it with ember-concurrency (we use it for the typeahead logic in power-select). The library offers cancellable/restartable async tasks. 
I understand we are under tight deadline, please add a comment to refactor + jira tickets for now. 

#### Doc:
http://ember-concurrency.com/docs/cancelation


#### Example in our code:
https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/pods/rca/controller.js#L53

Reply:
Great idea. Thanks. I prefer to implement it now.

Reply:
Yes, thanks. I prefer to refactor this now.